_proposals/pup-0001.rst

```diff
@@ -0,0 +1,328 @@
+proposal number. The proposal is a single file, e.g. `pup-0022.rst`. A core dev [#f1]_ is expected
+to merge any proposal with adequate clarity and in the expected format.
+
+
+Announcement
+^^^^^^^^^^^^
+
+Once a proposal is merged to the proposals area, an announcement would be sent to the appropriate
+list, which would include a very brief introduction and a link to the proposal in its web viewable
+form.
+
+
+Discussion and Feedback
+^^^^^^^^^^^^^^^^^^^^^^^
+
+General discussion would occur on the announcement thread on the appropriate mailing list. Threaded
+e-mail readers should make this conversation format easy to follow, while keeping the proposal
+document in-tact and easy to read.
+
+
+Revision
+^^^^^^^^
+
+Revision would occur as additional pull requests against the active proposal. This allows the
+changelog of a proposal to always be maintained. This is how Python does it. This also allows
+edits/updates to come with a commit message indicating the concerns addressed or improvements
+including links to responses via the archive. Collaboration is also easy with a collaborator sending
+revisions to the author as a patch or a new pull request against the proposal.
+
+
+Voting
+^^^^^^
+
+The +1, +0, -0, -1 feedback process is used, with votes sent as responses via the
+announcement/discussion mailing list. The voting has the following meanings:
+
+* +1: "Will benefit the project and should definitely be adopted."
+* +0: "Might benefit the project and is acceptable."
+* -0: "Might not be the right choice but is acceptable."
+* -1: "Not the right choice and should definitely not be adopted." This type of feedback must come
+  with a statement of objective concerns explaining what harm will come from adopting the proposal.
+
+Note that votes can come from anyone, not just core devs [#f1]_. This is directly modeled after [the
+Django process](https://docs.djangoproject.com/en/dev/internals/contributing/bugs-and-features/#how-we-make-decisions)
+who modeled their process after the Python and Apache communities.
+
+
+Deciding
+^^^^^^^^
+
+* If an obvious consensus arises via voting, the proposal is ratified. This is a bit vague, but it
+  works for the Django community.
+* Discussion on the thread can confirm if consensus is obvious or not.
+* Votes come from anyone, not just core devs [#f1]_.
+* A -1 from a core dev [#f1]_ is considered a veto vote and will block ratification until the vote
+  is recast.
+* Core devs are expected to consider and echo serious feedback from the community, especially any -1
+  votes from community members.
+* Consensus is not always possible. If obvious consensus is not reached, then the core devs [#f1]_
+  decide.
+
+
+Proposal Requirements
+=====================
+
+This is a modified version of [the Foreman template](https://github.com/theforeman/rfcs/blob/master/0000-template.md).
+
+::
+
+    ---
+    title: <title goes here>
+    author: <author or authors named>
+    ---
+
```

Comment:
Somewhere around here, maybe in the Summary block, maybe by using metadata at the top, should be a templated section that details the PUP status, like python PEPs. The section should include the current approval state, and could also include created/updated timestamps.

I'll happily mess with jekyll and put something together if you want me to, just let me know.